examples/with-tailwindcss/tailwind.config.js

```diff
@@ -1,12 +1,8 @@
 module.exports = {
-  purge: ['./components/**/*.{js,ts,jsx,tsx}', './pages/**/*.{js,ts,jsx,tsx}'],
-  darkMode: 'media', // 'media' or 'class'
+  purge: ['./pages/**/*.js', './components/**/*.js'],
```

Comment:
```suggestion
  purge: ['./pages/**/*.{js,ts,jsx,tsx}', './components/**/*.{js,ts,jsx,tsx}'],
```

Reply:
Opened a PR for the tailwind docs here: https://github.com/tailwindlabs/tailwindcss.com/pull/727